### DISCUSSION TITLE
Project and target settings configuration

### DISCUSSION THREAD

**Discussion by EastHell** [wojciech-kulik/xcodebuild.nvim#133] [Ideas]
Looks like xcodebuild is already good tool. 
At this point we can modify project.pbxproj by hands, but it's not that easy because here's a lot of keys and values, which hard to manage by hands. Is it possible to add a project settings configurator to plugin?

**Comment by wojciech-kulik** [comment]
I think `XcodeProj` can change build settings. I would see it as a configuration with the following steps:
- Select Target
- Select Build Configuration
- Select Predefined Build Setting
- Enter your value (prefilled with the current one)

However, in Xcode it's easier because possible values are available to select, I don't know if there is any global and importable list of possible values to use them in the plugin.

**Reply by EastHell** [reply]
Yes it can, but it require to be added to project as dependency. I simply can't do things like that with working projects. Main point is to manage project settings without not necessary dependencies and project generation files just out of the box if possible.

**Reply by wojciech-kulik** [reply]
No, no, I meant CLI tool for project management :) It is already used when you update files via nvim-tree/oil/neo-tree. It was more like a note to myself, what could be used to implement this feature.
